app/views/entities/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'devise/shared/navbar' %>
+<div class="group-new">
+  <%= form_with model: @entity, html: {'data-turbo' => "false"}, url: entities_path do |form| %>
+    <div class="form-new-title field">
+      <%= form.label :name %>
+      <%= form.text_field :name, required: true %>
+    </div>
+    <div class="form-new-post field">
+      <%= form.label :amount %>
+      <%= form.number_field :amount, class: "texto" %>
+    </div>
+    <div>
+      <%= form.hidden_field :user_id, required: true, value: current_user.id %>
+    </div>
+    <div class="field">
+      <%= form.label :category %>
+      <%= form.select :group_id, @groups.collect { |category| [category.name, category.id] }, {selected: params['id']}, class: "form-select texto"%>
+    </div>
+    <div class="actions">
+      <%= form.submit 'Create Transaction', class: 'form-submit-comment' %> 
+    </div>
```

Comment:
- [ ] Add new transaction button should be available only inside the category, Now transactions link is available on navbar, when I click on it I can create a transaction even though I do not select a category, then I get an error: 
<img width="455" alt="Screenshot 2022-06-03 at 22 24 13" src="https://user-images.githubusercontent.com/77038610/171923917-63f45774-ffdb-43aa-a1c1-e9501b3db915.png">

Kindly make sure that add transaction button is available only inside a categoty